about.html

```diff
@@ -0,0 +1,143 @@
+        </div>
+        <p class="intro">
+          The East African Developer summit brings together the community
+          of experts, academics, activits, consultant and businesses
+          who comprises of web development in a different country in educator
+          every year. This year, we hope to expand our invitation list 
+          including organisations and individuals
+        </p>
+        <p class="contact">
+          Please contact us per email for any further question
+          about EAC developer summit 2022!
+          <strong>eedcsummit2022@eabuja.org</strong>
+        </p>
+        <div class="bg-white">
+          <div class="summit-logo">
+            <div class="head">
+              <p class="main1">East Afria Developer Summit 2022 Logo</p>
+              <div class="red-rule"></div>
+            </div>
+            <p class="studies">The logo of East African Developper Summit was decided through the logo
+              competition from 4 April to 8 May.
+            </p>
+            <div class="logo-about">
+              <img src="images/logo1.png" class="logo" alt="my logo">
+            </div>
+          </div>
+          <div class="grey-rule"></div>
+          <div class="head">
+            <p class="main1">See the past EAC developer Summit</p>
+            <div class="red-rule"></div>
+          </div>
+          <p class="studies">Take a look at two past East africa developer summit which took place in Dar Es salaam and nairobi
+          </p>
+          <div class="overlays">
+            <div class="overlay">
+              <p class="year">2020</p>
+              <p class="main1">East African Developer Summit in Nairobi</p>
+            </div>
+            <div class="overlay1">
+              <p class="year">2020</p>
+              <p class="main1">East African Developer Summit in Kampala</p>
+            </div>
+          </div>
+        </div>
+        <section class="patners">
+          <div class="head">
+            <p class="main1">Partners</p>
+            <div class="red-rule"></div>
+          </div>
+          <div class="mypatners">
+            <img class="patner-img" src="./images/partners/google.png" alt="google">
+            <img class="patner-img" src="./images/partners/daumkakao.png" alt="daumkakao">
+            <img class="patner-img" src="./images/partners/microsoft.png" alt="microsoft">
+            <img class="patner-img" src="./images/partners/mozilla.png" alt="mozilla">
+            <img class="patner-img" src="./images/partners/airbnb.png" alt="airbnb">
+            <img class="patner-img" src="./images/partners/naver.png" alt="naveer">
+          </div>
+        </section>
+      </section>>
+    </div>
+    
+    <footer>
+      <div class="footer">
+        <img class="logo" src="./images/logo1.png" alt="my logo">
+        <p class="main1"> 2022 East African Developper Bujumbura some rights reserved</p>
+      </div>
+    </footer>
```

Comment:
- [ ] Please kindly ensure that the footer for the about page in  the desktop view is dark according to the original design.
- [ ] Please also ensure that the size of the logo in the footer matches the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169177386-78e5a972-e1cd-4aac-9a2e-cd99ba823dc0.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169177308-95807856-128b-4286-b6e4-4dd42cf8150c.png)